### PR TITLE
Moved Bootstrap dll to fix VS/AnyCPU projects

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Moved the dll from runtimes\lib\native\PLATFORM to runtimes\win-PLATFORM\native

Fixes https://task.ms/33477857, #1217 and maybe #893

### CODE REVIEW

File: build/CopyFilesToStagingDir.ps1 [hunk 1]
```diff
@@ -129,9 +129,9 @@ PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.ApplicationMode
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.System.winmd $NugetDir\lib\uap10.0
 PublishFile $FullBuildOutput\WindowsAppSDK_DLL\Microsoft.Windows.System.Power.winmd $NugetDir\lib\uap10.0
 #
-# Native (not managed, no MSIX)
-PublishFile $FullBuildOutput\WindowsAppSDK_BootstrapDLL\Microsoft.WindowsAppSDK.Bootstrap.dll $NugetDir\runtimes\lib\native\$Platform
-PublishFile $FullBuildOutput\WindowsAppSDK_BootstrapDLL\Microsoft.WindowsAppSDK.Bootstrap.pdb $NugetDir\runtimes\lib\native\$Platform
+# Native (not managed, AppLocal / no MSIX)
+PublishFile $FullBuildOutput\WindowsAppSDK_BootstrapDLL\Microsoft.WindowsAppSDK.Bootstrap.dll $NugetDir\runtimes\win-$Platform\native
```
Comment (AdamBraden) [inline build/CopyFilesToStagingDir.ps1:133]: should this be win10-$Platform?

Comment (DrusTheAxe) [reply build/CopyFilesToStagingDir.ps1:133]: Doh! Copy/paste error. Thanks!

**Comment by jonwis** [comment]
Can you double-check that building an app with the new results works in the way you expect? Is this hidden from developers who use the packages, or is it something @MikeHillberg  should be tracking?

**Review by jonwis — APPROVED** [review]

**Review by BenJKuhn — APPROVED** [review]

**Comment by DrusTheAxe** [comment]
> Can you double-check that building an app with the new results works in the way you expect?

@AdamBraden has tests to verify the scenarios, just awaiting a build with the fix to verify.

> Is this hidden from developers who use the packages

Yes. Just sausage in how we implement the NuGet. There's some spurs or spinoffs from this initial issue (e.g. making a class library vs app) that @AdamBraden is following up on.

**Comment by huichen123** [comment]
> > Can you double-check that building an app with the new results works in the way you expect?
> 
> @AdamBraden has tests to verify the scenarios, just awaiting a build with the fix to verify.
> 
> > Is this hidden from developers who use the packages
> 
> Yes. Just sausage in how we implement the NuGet. There's some spurs or spinoffs from this initial issue (e.g. making a class library vs app) that @AdamBraden is following up on.

we also need to fix WindowsAppSDK-Nuget-Native.C.props.

**Comment by huichen123** [comment]
> > Can you double-check that building an app with the new results works in the way you expect?
> 
> @AdamBraden has tests to verify the scenarios, just awaiting a build with the fix to verify.
> 
> > Is this hidden from developers who use the packages
> 
> Yes. Just sausage in how we implement the NuGet. There's some spurs or spinoffs from this initial issue (e.g. making a class library vs app) that @AdamBraden is following up on.

do we still need Microsoft.WindowsAppSDK.Bootstrap.targets?

**Pull request merged**